### PR TITLE
Feature/ch1241/pre qa healthcheck

### PR DESCRIPTION
**Pull request by SamMed1**

### CODE REVIEW

File: _includes/assets/css/styles.css [hunk 1]
```diff
@@ -401,7 +403,7 @@ main ul, main ol {
 COMPONENTS
 ========================================================================== */
 .title-bar {
-  background-color: #CE132D;
+  background-color: green;
```
Comment (DarkSnow) [inline _includes/assets/css/styles.css]: Should this be a HEX code or a variable? Green will be different in every browser.

File: _includes/assets/sass/_components.scss [hunk 2]
```diff
@@ -2,646 +2,13 @@
...
+@import "components/title-bar";
+@import "components/leader-bar";
+@import "components/header-block";
+@import "components/tiles-block";
+@import "components/image-l-text-r-block";
+@import "components/footer";
+@import "components/landing-block";
+@import "components/breadcrumbs";
+@import "components/carousel";
+@import "components/team-block";
```
Comment (DarkSnow) [inline _includes/assets/sass/_components.scss]: Very good. This file was massive for no good reason :)

Newline needed.

File: _includes/assets/sass/_typography.scss [hunk 3]
```diff
@@ -116,4 +120,20 @@ h4, .h4-style {
...
+	@media only screen and (min-width : $tablet) {
+		font-size: $lg-body-font-size;
+		line-height: $lg-body-line-height;
+	}
+}
+
+// Colours
+.white-text {
+	color: $white;
 }
```
Comment (DarkSnow) [inline _includes/assets/sass/_typography.scss]: Blank line at the end of files is part of the Drupal coding standards. I don't really care as long as it works, but it does get rid of these annoying messages.

File: _includes/assets/js/inline.js [hunk 4]
```diff
@@ -50,5 +50,17 @@ $(document).ready(function(){
...
+	
+	tdiv1 = div1.clone();
+	tdiv2 = div2.clone();
+	
+	if(!div2.is(':empty')){
+		div1.replaceWith(tdiv2);
+		div2.replaceWith(tdiv1);	  
+	}
+
 });
```
Comment (DarkSnow) [inline _includes/assets/js/inline.js]: Newline needed?

File: _includes/assets/css/styles.css (hunk 1 above)
Comment (SamMed1) [reply _includes/assets/css/styles.css]: Argh, this was test css that I hadn't removed. Cheers!

File: _includes/assets/sass/_components.scss (hunk 2 above)
Comment (SamMed1) [reply _includes/assets/sass/_components.scss]: Cheers, newline added.

File: _includes/assets/sass/_typography.scss (hunk 3 above)
Comment (SamMed1) [reply _includes/assets/sass/_typography.scss]: Great, totally makes sense. Added.

File: _includes/assets/js/inline.js (hunk 4 above)
Comment (SamMed1) [reply _includes/assets/js/inline.js]: Amended

**Pull request merged**